addons/s3/utils.py

```diff
@@ -122,3 +122,24 @@ def get_bucket_location_or_error(access_key, secret_key, bucket_name):
         return connect_s3(access_key, secret_key).get_bucket(bucket_name, validate=False).get_location()
     except exception.S3ResponseError:
         raise InvalidFolderError()
+
+
+def get_bucket_prefixes(access_key, secret_key, prefix, bucket_name):
+    bucket = connect_s3(access_key, secret_key).get_bucket(bucket_name)
```

Comment:
Here was were I introduced the bug, this function is no longer configured correctly. Here's a PR with the fix. https://github.com/CenterForOpenScience/osf.io/pull/10417 @cslzchen